lib/wechat/concern/common.rb

```diff
@@ -93,6 +93,12 @@ def wxa_get_user_phone_number(code)
         post 'business/getuserphonenumber', JSON.generate(code: code), base: Wechat::Api::WXA_BASE
       end
 
+      def wxa_get_user_risk_rank(openid:, scene:, **payload)
```

Comment:
Could you change method signature, maybe like `wxa_get_user_risk_rank(openid, scene = 1, **payload)`? 

Because all the method signature in this file using traditional parameter (with default value) instead of named parameter.